modules/_labs/select/react/lib/SelectOption.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {styled} from '@workday/canvas-kit-labs-react-core';
+import {keyframes} from '@emotion/core';
+import {
+  borderRadius,
+  colors,
+  commonColors,
+  // spacing,
+  type,
+  typeColors,
+} from '@workday/canvas-kit-react-core';
+// import { SystemIcon } from '@workday/canvas-kit-react-icon';
+// import { checkSmallIcon } from '@workday/canvas-system-icons-web';
+
+export interface SelectOptionProps extends React.LiHTMLAttributes<HTMLLIElement> {
+  value: string;
+  label: string;
+  disabled: boolean;
+  focused?: boolean;
+  selected?: boolean;
+  justSelected?: boolean;
+}
+
+const flashAnimation = keyframes`
+  0%, 49% {
+    background-color: ${commonColors.focusBackground};
+    color: ${typeColors.inverse};
+  }
+  50%, 100% {
+    background-color: transparent;
+    color: inherit;
+  }
+`;
```

Comment:
The flashing animation is a bit harsh:

<details>
    <summary>Warning: Flashing animation</summary>
    
![select-flashing](https://user-images.githubusercontent.com/338257/74264624-2f57c580-4cbe-11ea-8a96-c9ea1051fe23.gif)

</details>

Reply:
Yep, I tried to approximate the behavior of the native select, but looking at it again it is a bit much :). Timings/colors probably need to be adjusted.

Either way, I'm coordinating with Design on what the selection animation should be (this was just a guess, the final animation may be quite different).